garage/experiment/local_tf_runner.py

```diff
@@ -226,6 +324,8 @@ def train(self,
             plot: Visualize policy by doing rollout after each epoch.
             store_paths: Save paths in snapshot.
             pause_for_plot: Pause for plot.
+            _start_epoch: (internal) The starting epoch.
```

Comment:
this is not very Pythonic.

if you want a private API, create one. For instance you can have a private _train and that will allow to to specify start_epoch. The public train can depend on the private one.

Reply:
Make sense.